api/src/main/java/com/codeforcommunity/dto/school/School.java

```diff
@@ -47,4 +68,20 @@ public Country getCountry() {
   public void setCountry(Country country) {
     this.country = country;
   }
+
+  public Boolean getHidden() {
+    return hidden;
+  }
+
+  public void setHidden(Boolean hidden) {
+    this.hidden = hidden;
+  }
+
+  public List<SchoolContact> getContacts() {
+    return contacts;
+  }
+
+  public void setContacts(List<SchoolContact> contacts) {
+    this.contacts = contacts;
```

Comment:
Similar to the previous post, should we set this.contacts equal to a copy of the passed contacts?